.github/mergify.yml

```
@@ -88,8 +88,12 @@ shared:
   # Checks are passing
   checks_are_passing: &checks_are_passing
     - and:
-      - "#check-pending=0"
       - "#status-failure=0"
+      - or:
+        - "#check-pending=0"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)